process/errors.go

```diff
@@ -5,3 +5,5 @@ import "errors"
 var errNilTransactionHandler = errors.New("nil transaction handler provided")
 
 var errNilShardBlocksHandler = errors.New("nil shard blocks handler provided")
+
+var errNilEpochStartInfoHandler = errors.New("nil epoch start info handler provided")
```

Comment:
errNilEpochStartInfoProcessorHandler

Reply:
Each interface in this project is simply called **handler**, while the implementation of the handlers are called **processors**.

I would keep it as it is. Although, if you find this convention confusing, I can change all of them in this PR

Reply:
ok

Reply:
ok